src/Printer/Util.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Povils\PHPMND\Printer;
+
+/**
+ * @internal
+ */
+abstract class Util
+{
+
+    /**
+     * Get the snippet and information about it
+     *
+     * @param string $content
+     * @param int $line
+     * @param int|string $text
+     * @return array{snippet: string, line: int, magic: int|string, col: false|int}
+     */
+    final public static function getSnippet(string $content, int $line, $text): array
+    {
+        $content = str_replace(array("\r\n", "\r"), "\n", $content);
+        $lines = explode("\n", $content);
+
+        $lineContent = array_slice($lines, $line-1, 1);
+        $lineContent = reset($lineContent);
+        $start = strpos($lineContent, $text.'');
```

Comment:
This would be more clear as a string cast IMO

Reply:
I did not coding this method originally, this method was extracted current Printer\Xml class.
But I agree your opinion. I fixed with fb64a9b. Thanks pointed!